src/main/webapp/script.js

```diff
@@ -12,17 +12,42 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/** Fetches tasks from the server and adds them to the DOM. */
+function loadEntries() {
+  fetch('/list-entries').then(response => response.json()).then((entries) => {
+    const entryListElement = document.getElementById('entry-list');
+    entries.forEach((entry) => {
+      entryListElement.appendChild(createEntryElement(entry));
+    })
+  });
+  console.log("loaded entries!")
```

Comment:
It might also be helpful to print the number of entries that currently exist? Just an idea.

Reply:
Sounds good, i'll try adding that in the next sprint